azure/datalake/store/__init__.py

```diff
@@ -6,7 +6,7 @@
 # license information.
 # --------------------------------------------------------------------------
 
-__version__ = "0.0.1"
+__version__ = "0.0.2"
```

Comment:
Because I didn't rev the version during the previous PR and this is the version I will want to have published. Should I do this in a separate PR?